Carmunication/Controllers/SessionStart/SessionStartViewController.swift

```diff
@@ -214,6 +219,114 @@ extension SessionStartViewController {
             sessionStartView.journeyTogetherButton.layer.cornerRadius = buttonHeight / 2
         }
     }
+
+    private func sendPush() {
+        let functions = Functions.functions()
+        guard let databasePath = User.databasePathWithUID else {
+            return
+        }
+
+        // 유저의 친구 관계 리스트를 불러온다.
+        readUserFriendshipList(databasePath: databasePath) { friendshipList in
+            guard let friendshipList = friendshipList else {
+                return
+            }
+            // 친구 관계 id값으로 친구의 uid를 받아온다.
+            for friendshipID in friendshipList {
+                self.getFriendUid(friendshipID: friendshipID) { friendID in
+                    guard let friendID = friendID else {
+                        return
+                    }
+                    // 친구의 uid값으로 친구의 User객체를 불러온다.
+                    self.getFriendUser(friendID: friendID) { friend in
+                        guard let friend = friend else {
+                            return
+                        }
+                        self.friendDeviceToken.append(friend.deviceToken)
+                        print("친구들 Device Token : ", self.friendDeviceToken)
+
+                        // Functions 호출
+                        functions
+                            .httpsCallable("push_send")
+                            .call(["tokens": self.friendDeviceToken]) { (result, error) in
+                            if let error = error {
+                                print("Error calling Firebase Functions: \(error.localizedDescription)")
+                            } else {
+                                if let data = (result?.data as? [String: Any]) {
+                                    print("Response data -> ", data)
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+// MARK: - Firebase Realtime Database DB 관련 메서드
+extension SessionStartViewController {
+
+    // MARK: - DB에서 유저의 friendID 목록을 불러오는 메서드
+    private func readUserFriendshipList(databasePath: DatabaseReference, completion: @escaping ([String]?) -> Void) {
+        databasePath.child("friends").getData { error, snapshot in
+            if let error = error {
+                print(error.localizedDescription)
+                completion(nil)
+                return
+            }
+            let friends = snapshot?.value as? [String]
+            completion(friends)
+        }
+    }
+    // MARK: - friendID 값으로 DB에서 Friendship의 친구 id를 불러오는 메서드
```

Comment:
메서드간에는 개행을 하나 넣어주세요 !

```suggestion
    }

    // MARK: - friendID 값으로 DB에서 Friendship의 친구 id를 불러오는 메서드
```